galaxy_ng/app/migrations/0030_update_collection_remote_rhcertified_url.py

```diff
@@ -0,0 +1,30 @@
+from django.db import migrations
+
+def update_collection_remote_rhcertified_url(apps, schema_editor):
```

Comment:
Can you add a migration unit test to https://github.com/ansible/galaxy_ng/tree/master/galaxy_ng/tests/unit/migrations?

One test can be to create a remote named `rh-certified` with url `https://cloud.redhat.com/api/automation-hub/content/1237261-synclist/`, call the migration, and afterwards assert that the url is `https://console.redhat.com/api/automation-hub/content/1237261-synclist/`

You could also check that if it is named something else like `https://console.stage.redhat.com/api/automation-hub/` that it does not change